tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2492,3 +2492,900 @@ def max_call(x):
+            pytest.mark.c]
+        )
+    )
+)
+def test_full_like_basic_bool(language):
+    @types('int')
+    def create_full_like_shape_1d(n):
+        from numpy import full_like, shape
+        a = full_like([5,4,6,2], n, int, 'F')
+        s = shape(a)
+        return len(s),s[0]
+    @types('int')
+    def create_full_like_shape_2d(n):
+        from numpy import full_like, shape
+        a = full_like([[4,5,3,2],[7,4,2,1]], n, int, 'F')
+        s = shape(a)
+        return len(s),s[0], s[1]
+    @types('bool')
+    def create_full_like_val(val):
+        from numpy import full_like
+        a = full_like([4,5,6,7] , 3, bool, 'F')
+        return a[0],a[1],a[2]
+    @types('bool')
+    def create_full_like_arg_names(val):
+        from numpy import full_like
+        a = full_like([[4,3,6,6],[4,5,6,6]] ,fill_value = val, dtype=bool, shape = (2,3))
+        return a[0,0],a[0,1],a[0,2],a[1,0],a[1,1],a[1,2]
+
+    size = randint(10)
+    val  = bool(randint(2))
+
+    f_shape_1d  = epyccel(create_full_like_shape_1d, language = language)
+    assert(f_shape_1d(size)     == create_full_like_shape_1d(size))
+
+    f_shape_2d  = epyccel(create_full_like_shape_2d, language = language)
+    assert(f_shape_2d(size)     == create_full_like_shape_2d(size))
+
+    f_val       = epyccel(create_full_like_val, language = language)
+    assert(f_val(val)           == create_full_like_val(val))
+    assert(type(f_val(val)[0])       == type(create_full_like_val(val)[0].item()))
+
+    f_arg_names = epyccel(create_full_like_arg_names, language = language)
+    assert(f_arg_names(val)     == create_full_like_arg_names(val))
+    assert(type(f_arg_names(val)[0]) == type(create_full_like_arg_names(val)[0].item()))
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="arrays not implemented"),
+            pytest.mark.c]
+        )
+    )
+)
+def test_full_like_order(language):
+    @types('int')
+    def create_full_like_shape_C(n):
+        from numpy import full_like, shape
+        a = full_like([[6,4,5,1,3],[6,4,5,1,3]] ,4, order = 'C')
+        s = shape(a)
+        return len(s),s[0], s[1]
+    @types('int')
+    def create_full_like_shape_F(n):
+        from numpy import full_like, shape
+        a = full_like([[4,5,3,1,2],[6,4,5,1,3]],4, order = 'F')
+        s = shape(a)
+        return len(s),s[0], s[1]
+
+    size = randint(10)
+
+    f_shape_C  = epyccel(create_full_like_shape_C, language = language)
+    assert(f_shape_C(size) == create_full_like_shape_C(size))
+
+    f_shape_F  = epyccel(create_full_like_shape_F, language = language)
+    assert(f_shape_F(size) == create_full_like_shape_F(size))
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="casting to complex in not handled correctly"),
```

Comment:
Is there an issue for this?

Reply:
i didn't find any issue related to this,

Reply:
Then please can you create one

Reply:
sure.

Reply:
Could reference the issue in the reason please. E.g:
```
pytest.mark.skip(reason="Casting to complex is not handled correctly. See issue #723 ")
```